pkg/ccl/changefeedccl/changefeedbase/settings.go

```diff
@@ -349,3 +349,13 @@ var Quantize = settings.RegisterDurationSettingWithExplicitUnit(
 	time.Duration(metamorphic.ConstantWithTestRange("changefeed.resolved_timestamp.granularity", 1, 0, 10))*time.Second,
 	settings.DurationWithMinimum(0),
 )
+
+// MaxRetryBackoff is the maximum time a changefeed will backoff when in
+// a top-level retry loop, for example during rolling restarts.
+var MaxRetryBackoff = settings.RegisterDurationSettingWithExplicitUnit(
+	settings.ApplicationLevel,
+	"changefeed.max_retry_backoff",
+	"the maximum time a changefeed will backoff when retrying after a restart",
+	10*time.Minute, /* defaultValue */
+	settings.DurationWithMinimum(1*time.Second),
```

Comment:
Should we also set a upper bound? Maybe an hour?

Reply:
Interestingly, it doesn't override the first setting, but the error messages don't match the expected ranges. Fixed this, thanks for the tip.